rest-api/resources/src/main/resources/openapi/user/user.yaml

```diff
@@ -111,17 +111,6 @@ components:
             userId:
               allOf:
                 - $ref: '../openapi.yaml#/components/schemas/kapuaId'
-            mfaSecretKey:
-              type: string
-            trustKey:
-              description: |
-                The key of the trusted machine.
-              type: string
-            trustExpirationDate:
-              description: |
-                The moment when the machine trust will expire.
-              type: string
-              format: 'date-time'
```

Comment:
`trustKey` and `trustExpirationDate` are not involved in this PR, so the documentation about them should not be changed. You're probably not seeing them in your responses because you didn't activate any trusted machine, so they're null and as such not shown.